### DISCUSSION TITLE
Incorrect title?

### DISCUSSION THREAD

**Discussion by exadeci** [photoprism/photoprism#2357] [General]
Hello,

So I've just started using PhotPrism and my pictures are organised in folders by year, so I have "/Photos 2005" for example but in Photoprism the Title is shown as "<first 2 words of the picture name> 2022"

How is that generated? Can I get it to re-run? Can I just hide the details of the pictures?

**Comment by lastzero** [comment]
Can you add a screenshot? Depending on what section you are, album titles get generated based on the actual content. Also, you can rename them in many cases, should there be a mistake.

**Reply by exadeci** [reply]
@lastzero Here's an example
<img width="934" alt="Screen Shot 2022-05-26 at 1 05 42 pm" src="https://user-images.githubusercontent.com/1394130/170476240-de0e35a4-7f75-4117-adb4-ffff10a08962.png">

**Reply by lastzero** [reply]
What does the file metadata say? Does it have any?

**Reply by exadeci** [reply]
@lastzero No date in the exif aside from the last opened
<img width="257" alt="Screen Shot 2022-05-26 at 2 16 16 pm" src="https://user-images.githubusercontent.com/1394130/170486053-9457a38c-fdf0-4a62-9949-5888d71c9672.png">

**Reply by lastzero** [reply]
In that case, the year is likely not recognized correctly as the folder name is "Photos 2005" and not unambiguously "2005" AND the image includes no proper metadata.

**Reply by exadeci** [reply]
Of course, I understand. I'm just surprised it sets a date when it can't find the real one.
